src/main/pages/che-7/end-user-guide/assembly_the-developer-workspace-button.adoc

```diff
@@ -0,0 +1,46 @@
+---
+title: The Developer Workspace Button
+keywords:
+tags: []
+sidebar: che_7_docs
+permalink: che-7/the-developer-workspace-button
```

Comment:
@tomgeorge I suspect that you need a trailing slash here.

Reply:
https://github.com/eclipse/che-docs/pull/1462